llama_index/readers/twitter.py

```diff
@@ -28,6 +32,19 @@ def __init__(
         """Initialize with parameters."""
         self.bearer_token = bearer_token
         self.num_tweets = num_tweets
```

Comment:
very nit: don't need to set attributes anymore?

Reply:
Actually it was there before I started lol